libs/product/state/src/selectors/product/product.selectors.ts

```diff
@@ -50,38 +32,19 @@ const createProductPageSelectors = <T extends DaffProduct = DaffProduct>(): Daff
     selectProductState,
   } = getDaffProductFeatureSelector<T>();
 
-  const selectSelectedProductState = createSelector(
+  const selectCurrentProductState = createSelector(
     selectProductState,
     (state: DaffProductReducersState<T>) => state.product,
   );
 
-  const selectSelectedProductId = createSelector(
-    selectSelectedProductState,
-    (state: DaffProductReducerState) => state.selectedProductId,
-  );
```

Comment:
I don't think we want to remove this. Despite the fact that it was deprecated, its useful for URL-resolved product pages. Same for `selectSelectedProduct`

Reply:
changed